plugins/repository-locator/impl/spoon/src/main/java/org/pentaho/kettle/repository/locator/impl/spoon/SpoonRepositoryProvider.java

```diff
@@ -20,19 +20,19 @@
 import org.pentaho.di.core.service.ServiceProviderInterface;
 import org.pentaho.di.repository.Repository;
 import org.pentaho.di.ui.spoon.Spoon;
-import org.pentaho.kettle.repository.locator.api.KettleRepositoryLocator;
 import org.pentaho.kettle.repository.locator.api.KettleRepositoryProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.function.Supplier;
-import java.util.logging.Logger;
 
 /**
  * Created by bryan on 4/15/16.
  */
 @ServiceProvider( id = "SpoonRepositoryProviderProvider",
   description = "Get the repository from the provided PentahoSession", provides = KettleRepositoryProvider.class )
 public class SpoonRepositoryProvider implements KettleRepositoryProvider, ServiceProviderInterface<KettleRepositoryProvider> {
-  private static Logger log = Logger.getLogger( SpoonRepositoryProvider.class.getName() );
+  private Logger log = LoggerFactory.getLogger( SpoonRepositoryProvider.class.getName() );
```

Comment:
The common standard to declare Logger should be **private static final** Logger. Could exist some exceptions to this if you want to use it through inheritance, but does not seem to be the case.